drivers/storage/portworx/portworx_test.go

```diff
@@ -990,8 +990,12 @@ func TestStorageClusterDefaultsForCSI(t *testing.T) {
 	require.True(t, pxutil.IsCSIEnabled(cluster))
 	require.NotEmpty(t, cluster.Status.DesiredImages.CSIProvisioner)
 
+	// Feature gate should be removed when CSI is enabled
```

Comment:
nit: Did you mean feature gate should not be set?

Reply:
yeah, will fix